### PR TITLE
Fixed ggus_alerting.go

### PR DESCRIPTION
**Pull request by indrarahul2013**

### CODE REVIEW

File: src/go/MONIT/ggus_alerting.go [hunk 1]
```diff
@@ -273,9 +277,15 @@ func deleteAlerts() {
 		temp.Annotations.Scope = each.Annotations.Scope
 		temp.Annotations.URL = each.Annotations.URL
 
-		temp.StartsAt = each.StartsAt
+		emp.StartsAt = each.StartsAt
```
Comment (vkuznet) [inline src/go/MONIT/ggus_alerting.go:280]: are you sure that `emp` is an appropriate name here?

Comment (indrarahul2013) [reply src/go/MONIT/ggus_alerting.go]: No, it was a mistake.

Comment (vkuznet) [reply src/go/MONIT/ggus_alerting.go]: next time try to compile your code before submitting the PR since the compiler will fetch this kind of typos.

Comment (indrarahul2013) [reply src/go/MONIT/ggus_alerting.go]: Yes. I will follow this. 
Thanks

**Pull request merged**